tests/foreman/cli/test_system_registration.py

```diff
@@ -7,96 +7,58 @@ class SystemRegistrationTestCase(CLITestCase):
     """Tests for System Registration CLI"""
 
     @stubbed()
-    def test_rpm_install(self):
-        """@test: rpm can be retrieved
+    def test_positive_register_with_no_ak(self):
+        """@test: Register Content host to satellite without activation key
 
-        @feature: system registration
+        @feature: Content host
 
-        @assert: rpm required for modifying rhsm.conf can be installed from sat
-        host
-
-        @assert: rhsm.conf has been updated appropriately.
-
-        @status: Manual
-        """
-
-    @stubbed()
-    def test_register_no_ak(self):
-        """@test: register system to sat without activation key
-
-        @feature: system registration
-
-        @assert: system successfully registered to appropriate org
-
-        @status: Manual
-        """
-
-    @stubbed()
-    def test_register_with_ak(self):
```

Comment:
this test is removed since test_contenthost.py already has several tests which register content hosts with ak